glean-core/android/src/test/java/mozilla/telemetry/glean/scheduler/MetricsPingSchedulerTest.kt

```diff
@@ -510,10 +511,15 @@ class MetricsPingSchedulerTest {
             val expectedValue = "canary"
             expectedStringMetric.set(expectedValue)
 
+            // Set an experiment active to verify it gets sent in the pings
+            Glean.setExperimentActive("test-experiment", "a")
```

Comment:
This should really be an integration test for our sample app, along with most of the tests in this file. Nothing you can do here, though - this whole thing would require its own bug. @travis79 would you kindly file it?

Reply:
[Bug filed](https://bugzilla.mozilla.org/show_bug.cgi?id=1653955)